via/services/google_drive.py

```diff
@@ -16,27 +16,36 @@
 LOG = getLogger(__name__)
 
 
-def translate_google_error(error):
+def translate_google_error(
+    error,
+):  # pylint:disable=too-many-return-statements
     """Get a specific error instance from the provided error or None."""
 
     # This isn't a requests exception we can get meaningful data from
     if not isinstance(error, HTTPError) or error.response is None:
         return None
 
+    status_code = error.response.status_code
+
     # Try and parse out the Google details in the format we've seen
     try:
         google_error = error.response.json()["error"]["errors"][0]
-    except (JSONDecodeError, KeyError):
+    except (JSONDecodeError, KeyError, TypeError):
```

Comment:
The `[0]` raises `TypeError` if `error.response.json()["error"]["errors"]` is not a list (for example maybe Google sometimes returns `null` here)